Description: fix soname for libraries
 Add SOVERSION setting to cmake files.
Origin: vendor, https://gitlab.archlinux.org/archlinux/packaging/packages/protobuf/-/blob/main/soversion.patch?ref_type=heads
Author: Antonio Rojas <arojas@archlinux.org>
Forwarded: no
Last-Update: 2023-12-26

---

Index: protobuf/cmake/libprotobuf-lite.cmake
===================================================================
--- protobuf.orig/cmake/libprotobuf-lite.cmake
+++ protobuf/cmake/libprotobuf-lite.cmake
@@ -37,6 +37,7 @@ if(protobuf_BUILD_SHARED_LIBS)
 endif()
 set_target_properties(libprotobuf-lite PROPERTIES
     VERSION ${protobuf_VERSION}
+    SOVERSION ${protobuf_VERSION_MINOR}
     OUTPUT_NAME ${LIB_PREFIX}protobuf-lite
     DEBUG_POSTFIX "${protobuf_DEBUG_POSTFIX}"
     # For -fvisibility=hidden and -fvisibility-inlines-hidden
Index: protobuf/cmake/libprotobuf.cmake
===================================================================
--- protobuf.orig/cmake/libprotobuf.cmake
+++ protobuf/cmake/libprotobuf.cmake
@@ -39,6 +39,7 @@ if(protobuf_BUILD_SHARED_LIBS)
 endif()
 set_target_properties(libprotobuf PROPERTIES
     VERSION ${protobuf_VERSION}
+    SOVERSION ${protobuf_VERSION_MINOR}
     OUTPUT_NAME ${LIB_PREFIX}protobuf
     DEBUG_POSTFIX "${protobuf_DEBUG_POSTFIX}"
     # For -fvisibility=hidden and -fvisibility-inlines-hidden
Index: protobuf/cmake/libprotoc.cmake
===================================================================
--- protobuf.orig/cmake/libprotoc.cmake
+++ protobuf/cmake/libprotoc.cmake
@@ -27,6 +27,7 @@ endif()
 set_target_properties(libprotoc PROPERTIES
     COMPILE_DEFINITIONS LIBPROTOC_EXPORTS
     VERSION ${protobuf_VERSION}
+    SOVERSION ${protobuf_VERSION_MINOR}
     OUTPUT_NAME ${LIB_PREFIX}protoc
     DEBUG_POSTFIX "${protobuf_DEBUG_POSTFIX}"
     # For -fvisibility=hidden and -fvisibility-inlines-hidden
